.github/workflows/test.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
     - name: Setup Go
       uses: actions/setup-go@v2
       with:
-        go-version: 1.16.7
+        go-version: 1.17.1
```

Comment:
Side note we had an issue with Nitric membrane project and Go 1.17, generating proto files?

https://github.com/envoyproxy/protoc-gen-validate/issues/508#issuecomment-913441731

Reply:
correct I remember troubleshooting that with you :). Haven't had an issue locally or in the CI. And the auto generated API client doesn't use this plugin and uses a different baseline version as well.